README.md

```diff
@@ -127,62 +127,87 @@ APP_NIGHTFALL_MNEMONIC=
 APP_TOKEN_ADDRESS_1=0x4f3c4F8D4575Cf73c2FAf9F36cc505e19E65B9C0
 ```
 
+```
+# Contents of .env.goerli
+APP_BLOCKCHAIN_WEBSOCKET_URL=your web3 socket provider
+APP_CLIENT_API_URL=http://localhost:8080
+APP_ETH_PRIVATE_KEY=your Ethereum private key
+APP_NIGHTFALL_MNEMONIC=your Nightfall mnemonic
+# Goerli MATIC
+APP_TOKEN_ADDRESS_1=0x499d11E0b6eAC7c0593d8Fb292DCBbF815Fb29Ae
+```
+
 ### Example scripts
 
+To use the example scripts and the SDK correctly, one needs to have a good understanding of how the Nightfall Protocol runs under the hood.
+
 #### 2Tx rule
 
-This is a rule that explains the current Nightfall Protocol on Ganache and Testnet. To learn about Nightfall protocol on Mainnet visit the [documentation](https://docs.polygon.technology/docs/nightfall/faq/#how-long-do-transfers-take-on-polygon-nightfall-network-from-start-to-finish).
-The rule applies to the example scripts below and it is stated here for clarity.
+<b>This is a rule that applies to Nightfall Protocol on Ganache.<b>
+
 Making a deposit, transfer or withdrawal means that a transaction is submitted to L2, when 2 transactions like this are submitted a block is proposed and created. The creation of a new block changes the state of Nightfall. Changing the state of L2 means that the deposit, transfer and withdrawal(not finalise-withdrawal) are finalised.
+ex. Making 1 deposit won't change the state of Nightfall. Running the `eg:nf-balances:ganache` script won't show any updated balance with the new deposit. Making 2 deposits or a deposit and a transfer will update the state and show the correct updated balance when running the script.
+
+#### 32Tx rule
+
+<b>This is a rule that applies to Nightfall Protocol on Goerli Testnet and Mainnet.<b>
+To learn more about Nightfall protocol on Mainnet and Testnet visit the [documentation](https://docs.polygon.technology/docs/nightfall/faq/#how-long-do-transfers-take-on-polygon-nightfall-network-from-start-to-finish).
+
+Making a deposit, transfer or withdrawal means that a transaction is submitted to L2, when 32 transactions like this are submitted a block is proposed and created. The creation of a new block changes the state of Nightfall. Changing the state of L2 means that the deposit, transfer and withdrawal(not finalise-withdrawal) are finalised.
+ex. Making 1 deposit won't change the state of Nightfall. Running the `eg:nf-balances:goerli` script won't show any updated balance with the new deposit. Another 31 transactions should be submitted to the network
```

Comment:
Note the reference to `balances` script.

Also, I see the point about repeating the same as in the `2Tx rule` section - to emphasise the behaviour, but I wonder if a shorter version would be enough :thinking: Up to you!!

Reply:
it is important to be clear to understand the difference, but yeah it is too big and repetitive, did a shorter version!